Makefile

```diff
@@ -317,6 +317,11 @@ bundle-build: bundle bundle-update ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+# Run bundle image
+.PHONY: bundle-run
+bundle-run: operator-sdk
+	$(OPERATOR_SDK) -n openshift-operators run bundle $(BUNDLE_IMG)
```

Comment:
> > > I am not sure that code that is intended purely for dev purposes belongs here, my first intuition is that this would belong in a more internal space.
> > 
> > 
> > That's a good point and I don't have strong opinion about it
> 
> @slintes , @razo7 WDYT, do you have any preference ?

I join some of us who I believe agree that this is kind of gray area, if this target is needed or in the right place. I don't mind having it. Maybe just the the change the bundle name to include something about that specific namespace, `-n openshift-operators` or even remove it from the target and set is as a variable that we can export?

Reply:
A variable we can export seems a good improvement